### PR TITLE
fix: Enable Extra Env in Chart

### PR DESCRIPTION
**Pull request by 3schwartz**
## Related Tickets & Documents

In [PR #487](https://github.com/Twingate/kubernetes-operator/pull/487), support was added to provide additional environment variables to the operator.

However, it is currently not possible to specify these variables via the Helm Chart.

## Changes

This PR enables the ability to provide extra environment variables to the operator from the Helm Chart.

### CODE REVIEW

**Comment by 3schwartz** [comment]
Hi @ekampf 

I realized that in [PR #487](https://github.com/Twingate/kubernetes-operator/pull/487), I forgot to make the new settings configurable through the Helm chart.
Would you be able to review this?

Should I also bump the chart version as part of this change?

**Comment by ekampf** [comment]
@3schwartz yeah missed that too...
yes Please up the chart version and I'll release another version

**Comment by 3schwartz** [comment]
> @3schwartz yeah missed that too... yes Please up the chart version and I'll release another version

@ekampf Done

One of the integration tests is failing with the error remote_network_id missing, but I’m not sure why my changes would cause this 🤔

The chart is currently referencing the latest image. Once this PR is merged, will both the new chart and the updated image be available with the changes from the prior PR, or is a release action required?

File: deploy/twingate-operator/templates/deployment.yaml [hunk 1]
```diff
@@ -70,6 +70,10 @@ spec:
           - name: TWINGATE_REMOTE_NETWORK_NAME
             value: {{ .Values.twingateOperator.remoteNetworkName }}
           {{- end }}
+          {{- range $key, $value := .Values.extraEnvVars }}
+          - name: {{ $key }}
```
Comment (ekampf) [inline deploy/twingate-operator/templates/deployment.yaml:74]: Do we want to do that or just keep the k8s structure (name, value) and do something like `{{- toYaml .Values.extraEnvVars | nindent 8 }}`

Comment (ekampf) [reply deploy/twingate-operator/templates/deployment.yaml:74]: I'm looking at whats common in other charts and it seems most do something like:
```
## extraEnvVars:
##   - name: FOO
##     value: "bar"
```

**Comment by ekampf** [comment]
@3schwartz closing this in favor of https://github.com/Twingate/kubernetes-operator/pull/491

**Pull request closed without merging**